Misc improvements

richardolsson opened the pull request:
This PR contains a number of smaller improvements to the Organize interface.

### Manage section
The manage section that was implemented in the last release but hidden, is now made visible.

### Welcome message on dashboard
As outlined in #730, a message is added on the dashboard that includes the name of the current organization.

![image](https://user-images.githubusercontent.com/550212/39088095-863e154a-45ac-11e8-857c-7f35a76bcf5a.png)

### Campaign playback
Fixes three issues related to the `CampaignPlayer`. One undocumented issue related to how the order in which actions were stored affected the playback span, one missing import in `CampaignPlaybackPane` and a third one as described in #658.

### Alert messages when unauthorized
Adds a framework for alert messages to display in the bottom left corner, and logic to display a message whenever an API request results in a 403 (unauthorized) error.

![image](https://user-images.githubusercontent.com/550212/39092815-ba5f04ec-4615-11e8-9e0c-1fa40b91b800.png)

Issues #745, #746, #747 and #748 describe future improvements to and use cases for this framework.

Bagera commented:
Looks good.
The only things I can find that could be a bit different are:
- AlertMessages-messageItem-inOutAnimation: The keyframes have some duplication and should probably use transform instead of setting different `top` to get smoother animation.
- The recalc of map bounds could be a function to avoid repeat

richardolsson commented:
Great feedback, @Bagera! I have optimized the CSS animation now.

The map bounds recalculation should not be identical in two places, but use different props, which it didn't. Your comment alerted me to this, and I have fixed it now. It could still be broken out into a function, but I don't think it's necessary for such a tiny repetition.

In the process I came across another bug and fixed that too.

Please review again and approve, if you think it's ok. :)

richardolsson commented:
Thank you! Merging. :)
